etc/settings/settings_test.json

```diff
@@ -220,27 +220,55 @@
             }
         },
         "Simulate RTL": {
+                "filepath": {
+                "label": "Waveform file:",
+                "widgetType": "input",
```

Comment:
if you use the "filepath" widgetType, the field will give you the [...] button for browsing files

Reply:
I guess this "widgetType": "input", gives we simple line edit. I don't see this button. But I'll change it

Reply:
Fixed

Reply:
I need "input". No need file path.